test/index.js

```diff
@@ -1,2 +1,3 @@
+require('any-promise/register')('bluebird', {Promise: require('bluebird')});
```

Comment:
The added comment looks good, along with the note in the readme about defaulting to window.Promise that you already added